### ISSUE TITLE
How to rename a prefix

### ISSUE THREAD

**Issue by mambolevis** [justbur/emacs-which-key#133]
Hi,

Is it possible to rename a `e -> +prefix` for something like this  `e -> elisp-prefix`.  I also want to modify the following prefixes for something more informative:
![image](https://cloud.githubusercontent.com/assets/13024208/17279146/d1e2a440-576d-11e6-8dab-50c2daa64a2b.png)
for example: `e -> elisp-prefix` , `m -> mode-prefix`  and `w -> windows-prefix`

Thanks
Regards,
Levis

**Comment by mambolevis** [comment]
I found a solution using: 
```
  (which-key-add-key-based-replacements
    "s-f e" "elisp"
    "s-f m" "modes-handling"
    "s-f w" "window")
```
Is it the right (only) way to do it?

**Comment by justbur** [comment]
Yes there are a couple of ways to do it but that one should be fine.

**Comment by mambolevis** [comment]
OK. Thanks.

**Comment by gzxiaoxiang** [comment]
I have one question on using which-key in minibuffer mode.   The Readme document indicate that 'Minibuffer Option' is useful by (which-key-setup-minibuffer) and (setq which-key-popup-type 'minibuffer) configuration, but we can not get the popup window in which-key minibuffer mode. The test environment is emacs24.5 and spacemacs. We should be appreciate for your help , thank you very much!

**Comment by justbur** [comment]
@gzxiaoxiang Should be fixed in 06444f3
